### DISCUSSION TITLE
getting-started/install/zotero

### DISCUSSION THREAD

**Discussion by giscus** [aidenlx/obsidian-zotero#90] [Announcements]
# getting-started/install/zotero

1. Download zotero-obsidian-note-*.xpi from GitHub Release:  (with tag )

https://obzt.aidenlx.top/getting-started/install/zotero

**Comment by GatoEN** [comment]
The Github Release button did not allow me to download the file, the browser popup claimed it's broken. Instead I could download the file at https://github.com/aidenlx/obsidian-zotero/releases/

**Comment by haiqiang-zhang** [comment]
The add-on "Obsidian Note for Zotero" could not be installed. It may be incompatible with this version of Zotero.
